### ISSUE TITLE
Accessibility: aria-labelledby: logo-facebook in footer

### ISSUE THREAD

**Issue by hackmastera** [pulibrary/lux#341]
There's an `aria-labelledby` but no element with the id `logo-facebook`. This causes a failure with the axe-core-rspec integration on pulfalight.

Selector: svg[aria-labelledby="logo-facebook"]

This appears to be a wcag2a violation.

Perhaps just use `aria-label` instead.
